### Build Failed

:x: Something went wrong while validating this pull request.

#### Build Commands

```sh
```

##### Stdout log
###### (last 100 lines)

```console
Config enriched: If no api token specified, use environment variable
Config enriched: If source control is github, set git binary path
Config enriched: Set the slackbot url
Config enriched: If github SourceControlType and null StatusUpdateType, use github
Config enriched: If github SourceControlType and StatusUpdaterType and null ApiToken, use the one from SourceRetriever
Config enriched: If github SourceControlType and StatusUpdaterType and null Organization, use the one from SourceRetriever
Config enriched: If github SourceControlType and StatusUpdaterType and null Repository, use the one from SourceRetriever
Config enriched: If github SourceControlType and StatusUpdaterType and null PullRequest, use the one from SourceRetriever
Config enriched: CheckstyleAnalyzer is enabled by default
Retrieving source with org.pentaho.build.buddy.bundles.source.github.SlackbotGithubSourceRetriever@609af454
VCS Commands:
clone: [git, clone, --depth=1, --branch, master, https://[GITHUB TOKEN]@github.com/pentaho/pentaho-engineering-samples.git, pentaho-engineering-samples-base-42-5b23c728-40c6-44dd-a5be-dbed1f56343c]
fetch: [git, fetch, --depth=2, origin, pull/42/head:pullRequest]
merge: [git, merge, --no-edit, --no-ff, -Xtheirs, -Xignore-space-change, pullRequest]
HEAD is now at d283a1b Merge pull request #37 from jonjarvis/BISERVER-13837
Merge made by the 'recursive' strategy.
 .../javascript-apis/platform/visual-samples-bar-d3/.npmignore       | 1 -
 .../javascript-apis/platform/visual-samples-bar-d3/sandbox.html     | 6 +++---
 2 files changed, 3 insertions(+), 4 deletions(-)
GithubSourceRetriever file list:
[modified]	Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/.npmignore
[modified]	Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/sandbox.html
BuildToolDetector unable to determine build type, falling back to config defaults
Build file exclusion pattern: (\A|/)(assembly|assemblies)/
```

#### Stderr log
###### (last 100 lines)

```console
Cloning into 'pentaho-engineering-samples-base-42-5b23c728-40c6-44dd-a5be-dbed1f56343c'...
From https://github.com/pentaho/pentaho-engineering-samples
 * [new ref]         refs/pull/42/head -> pullRequest
java.lang.Exception: java.lang.RuntimeException: org.eclipse.egit.github.core.client.RequestException: Not Found (404)
	at org.pentaho.build.buddy.bundles.orchestrator.OrchestratorImpl.orchestrate(OrchestratorImpl.java:387)
	at Proxydb7010ba_4c63_496c_a092_7bcd6ef97c89.orchestrate(Unknown Source)
	at org.pentaho.build.buddy.bundles.rest.OrchestratorRestService$1.write(OrchestratorRestService.java:62)
	at org.apache.cxf.jaxrs.provider.BinaryDataProvider.writeTo(BinaryDataProvider.java:172)
	at org.apache.cxf.jaxrs.utils.JAXRSUtils.writeMessageBody(JAXRSUtils.java:1381)
	at org.apache.cxf.jaxrs.interceptor.JAXRSOutInterceptor.serializeMessage(JAXRSOutInterceptor.java:244)
	at org.apache.cxf.jaxrs.interceptor.JAXRSOutInterceptor.processResponse(JAXRSOutInterceptor.java:120)
	at org.apache.cxf.jaxrs.interceptor.JAXRSOutInterceptor.handleMessage(JAXRSOutInterceptor.java:83)
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308)
	at org.apache.cxf.interceptor.OutgoingChainInterceptor.handleMessage(OutgoingChainInterceptor.java:83)
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:308)
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:121)
	at org.apache.cxf.transport.http.AbstractHTTPDestination.invoke(AbstractHTTPDestination.java:253)
	at org.apache.cxf.transport.servlet.ServletController.invokeDestination(ServletController.java:234)
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:208)
	at org.apache.cxf.transport.servlet.ServletController.invoke(ServletController.java:160)
	at org.apache.cxf.transport.servlet.CXFNonSpringServlet.invoke(CXFNonSpringServlet.java:180)
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.handleRequest(AbstractHTTPServlet.java:298)
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.doPost(AbstractHTTPServlet.java:217)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:707)
	at org.apache.cxf.transport.servlet.AbstractHTTPServlet.service(AbstractHTTPServlet.java:273)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:812)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:587)
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:287)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:80)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)
	at org.eclipse.jetty.server.Server.handle(Server.java:499)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:311)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)
	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:544)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: org.eclipse.egit.github.core.client.RequestException: Not Found (404)
	at org.pentaho.build.buddy.bundles.source.github.GithubSourceRetriever.onStop(GithubSourceRetriever.java:280)
	at org.pentaho.build.buddy.bundles.source.github.SlackbotGithubSourceRetriever.onStop(SlackbotGithubSourceRetriever.java:36)
	at Proxy9d06b07d_ca2a_4133_adf6_83b92691d70e.onStop(Unknown Source)
	at org.pentaho.build.buddy.bundles.orchestrator.OrchestratorImpl.doNotBuild(OrchestratorImpl.java:516)
	at org.pentaho.build.buddy.bundles.orchestrator.OrchestratorImpl.orchestrate(OrchestratorImpl.java:354)
	... 41 more
Caused by: org.eclipse.egit.github.core.client.RequestException: Not Found (404)
	at org.eclipse.egit.github.core.client.GitHubClient.createException(GitHubClient.java:552)
	at org.eclipse.egit.github.core.client.GitHubClient.sendJson(GitHubClient.java:643)
	at org.eclipse.egit.github.core.client.GitHubClient.post(GitHubClient.java:757)
	at org.pentaho.build.buddy.bundles.source.github.GithubCommitService$Delegate$1.post(GithubCommitService.java:50)
	at org.eclipse.egit.github.core.service.CommitService.createStatus(CommitService.java:426)
	at org.pentaho.build.buddy.bundles.source.github.GithubCommitService.createStatus(GithubCommitService.java:35)
	at org.pentaho.build.buddy.bundles.source.github.GithubSourceRetriever.onStop(GithubSourceRetriever.java:278)
	... 45 more

```

--- a/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/.npmignore
+++ b/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/.npmignore
@@ -1,3 +1,2 @@
 *.tgz
 sandbox.html
-sample-data/*

--- a/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/.npmignore
+++ b/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/.npmignore
@@ -1,2 +1,3 @@
 *.tgz
 sandbox.html
+sandbox-data.json

--- a/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/sandbox.html
+++ b/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/sandbox.html
@@ -76,14 +76,14 @@
           .then(function (view) {
             // Handle the execute action.
             view.on("pentaho/visual/action/execute", {
-              "do": function (event) {
-                alert("Executed " + event.action.dataFilter.contentKey);
+              "do": function (event, action) {
+                alert("Executed " + action.dataFilter.contentKey);
               }
             });
 
             // Handle the select action.
             view.on("pentaho/visual/action/select", {
-              "finally": function (event) {
+              "finally": function (event, action) {
                 document.getElementById("messages_div").innerText = "Selected: " + view.selectionFilter.contentKey;
               }
             });

--- a/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/sandbox.html
+++ b/Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/sandbox.html
@@ -76,14 +76,14 @@
           .then(function (view) {
             // Handle the execute action.
             view.on("pentaho/visual/action/execute", {
-              "do": function (action) {
-                alert("Executed " + action.dataFilter.contentKey);
+              "do": function (event) {
+                alert("Executed " + event.action.dataFilter.contentKey);
               }
             });
 
             // Handle the select action.
             view.on("pentaho/visual/action/select", {
-              "finally": function (action) {
+              "finally": function (event) {
                 document.getElementById("messages_div").innerText = "Selected: " + view.selectionFilter.contentKey;
               }
             });

